admin/app.js

```diff
@@ -149,6 +155,20 @@ const sessionOptions = {
     tableName: '[mtc_admin].[sessions]'
   })
 }
+
+const sessionOptions = {
+  name: 'mtc-admin-session-id',
+  secret: config.SESSION_SECRET,
+  resave: false,
+  rolling: true,
+  saveUninitialized: false,
+  cookie: {
+    maxAge: config.ADMIN_SESSION_EXPIRATION_TIME_IN_SECONDS * 1000,
+    httpOnly: true,
+    secure: secureCookie
```

Comment:
We can only have the secure flag in the cookie on https, can't we?  Otherwise won't local dev break?